frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -0,0 +1,334 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  Switch,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  ActionGroup,
+  Button,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+
+import {
+  createModalLauncher,
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+  ModalComponentProps,
+  ModalFooter,
+} from '@console/internal/components/factory/modal';
+import { apiVersionForModel } from '@console/kubevirt-plugin/integration-tests/tests/utils/selectors';
```

Comment:
this needs to be imported from `@console/internal/module/k8s` and not `kubevirt-plugin` plugin.

Reply:
Good catch!!